src/routes.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+use Io\Token\Proto\Common\Alias\Alias;
+use Tokenio\Config\TokenCluster;
+use Tokenio\Config\TokenEnvironment;
+use Tokenio\Config\TokenIoBuilder;
+use Tokenio\Http\Request\TokenRequest;
+use Tokenio\Http\Request\TokenRequestOptions;
+use Tokenio\Security\UnsecuredFileSystemKeyStore;
+
+class TokenSample
+{
+    const DEVELOPER_KEY = '4qY7lqQw8NOl9gng0ZHgT4xdiDqxqoGVutuZwrUYQsI';
+
+    /**
+     * @var UnsecuredFileSystemKeyStore
+     */
+    private $keyStore;
+
+    private $tokenIO;
+    private $member;
+
+    public function __construct()
+    {
+        $this->keyStore = new UnsecuredFileSystemKeyStore(__DIR__ . '/../keys/');
+
+        $this->tokenIO = $this->initializeSDK();
+        $this->member = $this->initializeMember();
+    }
+
+    private function initializeSDK()
+    {
+        $builder = new TokenIoBuilder();
+        $builder->connectTo(TokenCluster::get(TokenEnvironment::SANDBOX));
+        $builder->developerKey(self::DEVELOPER_KEY);
+        $builder->withKeyStore($this->keyStore);
+        return $builder->build();
+    }
+
+    private function initializeMember()
+    {
+        $memberId = $this->keyStore->getFirstMemberId();
+        if (!empty($memberId)) {
+            return $this->loadMember($memberId);
+        } else {
+            return $this->createMember();
+        }
+    }
+
+    private function loadMember($memberId)
+    {
+        return $this->tokenIO->getMember($memberId);
+    }
+
+    private function createMember()
+    {
+        $time = ceil(time() / 60 * 5); // Update member account every 5 minutes
+        $email = 'asphp-' . $time . '+noverify@example.com';
```

Comment:
Can we use random String instead of time please?
1. for consistency
2. if I create a member, removed the keys and try to run the sample again in 5 minutes, I won't be able to create the member since the alias is taken.